doc/api/console.md

```diff
@@ -250,6 +290,15 @@ If formatting elements (e.g. `%d`) are not found in the first string then
 [`util.inspect()`][] is called on each argument and the resulting string
 values are concatenated. See [`util.format()`][] for more information.
 
+### console.table([data][, ...args])
```

Comment:
I'd prefer `console.table(tabularData[, properties])` like the [IDL](https://console.spec.whatwg.org/#console-namespace) specifies.